xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -141,6 +141,7 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
   #if ("$!commentSelection" != "")
     #set($isAnnotation = true)
   #end
+  #set ($isUserComment = $services.model.resolveDocument($comment.author, 'current') == $xcontext.userReference)
```

Comment:
I think the 'user' (hint) resolver would be more appropriate in this case.

Reply:
I didn't change the existing code but if you think the `user` hint is better then we can try to use it.